website/docs/r/registry_module.html.markdown

```diff
@@ -37,20 +37,58 @@ resource "tfe_registry_module" "test-registry-module" {
 }
 ```
 
+Create private registry module without VCS:
+
+```hcl
+resource "tfe_organization" "test-organization" {
+  name  = "my-org-name"
+  email = "admin@company.com"
+}
+
+resource "tfe_registry_module" "test-private-registry-module" {
+  organization    = tfe_organization.test-organization.name
+  module_provider = "my_provider"
+  name            = "another_test_module"
+  registry_name   = "private"
+}
+```
+
+Create public registry module:
+
+```hcl
+resource "tfe_organization" "test-organization" {
+  name  = "my-org-name"
+  email = "admin@company.com"
+}
+
+resource "tfe_registry_module" "test-public-registry-module" {
+  organization    = tfe_organization.test-organization.name
+  namespace       = "terraform-aws-modules"
+  module_provider = "aws"
+  name            = "vpc"
+  registry_name   = "public"
+}
+```
+
 ## Argument Reference
 
 The following arguments are supported:
 
-* `vcs_repo` - (Required) Settings for the registry module's VCS repository. Forces a
-  new resource if changed.
+* `vcs_repo` - (Optional) Settings for the registry module's VCS repository. Forces a
+  new resource if changed. One of `vcs_repo` or `module_provider` is required.
+* `module_provider` - (Optional) The provider of the registry module.
```

Comment:
I think we should be a little more specific about what input is expected here, since this appears to be a key attribute when vcs is not used. I was slightly confused until I thought more about it and looked at the docs.

```suggestion
* `module_provider` - (Optional) Specifies the Terraform provider that this module is used for. For example, "aws"
```